accord-core/src/main/java/accord/utils/async/AsyncChainCombiner.java

```diff
@@ -165,4 +165,26 @@ static <V> boolean canAppendTo(AsyncChain<? extends V> chain, BiFunction<V, V, V
             return reduce.reducer == reducer;
         }
     }
+
+    static class ReduceWithZero<A, B> extends AsyncChainCombiner<A, B>
+    {
+        private final B zero;
+        private final BiFunction<B, ? super A, B> reducer;
+
+        protected ReduceWithZero(List<? extends AsyncChain<? extends A>> inputs, B zero, BiFunction<B, ? super A, B> reducer)
```

Comment:
assuming this isn't removed before commit, could we call this ReduceWithInitialValue, or maybe ReduceWithIdentiity?

Reply:
renamed and updated variables